src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4CyclesPayoutsPanel/hooks/useV4ExportSplitsToCsv.ts

```diff
@@ -18,10 +17,10 @@ const CSV_HEADER = [
   'hook',
 ]
 
-const splitToCsvRow = (split: V4Split) => {
+const splitToCsvRow = (split: JBSplit) => {
   return [
     split.beneficiary,
-    `${new SplitPortion(split.percent).formatPercentage()}%`,
+    `${split.percent.formatPercentage()}%`,
```

Comment:
The advantage is that we can do stuff like this.

Reply:
Very nice